cmd/wallet-web/src/router/routes.js

```diff
@@ -60,6 +60,19 @@ export default [
       },
     ],
   },
+  {
+    path: 'oidc-share',
```

Comment:
Shouldn't this be just `oidc` ?

Reply:
> Fixing this in a future PR

We earlier agreed to create an issue and add a corresponding TODO in these cases @victro 
cc @rolsonquadras

Reply:
New issue created for this one:
https://github.com/trustbloc/wallet/issues/1562